packages/clerk-js/src/ui/common/authPropHelpers.ts

```diff
@@ -21,25 +21,24 @@ type ExtractAuthPropOptions =
     );
 
 /**
- *
- * extractAuthProp(key, options)
  *
  * Retrieve auth redirect props passed through querystring parameters, component props
  * or display config settings.
  *
  * The priority is:
- * Querystring parameters > ClerkJS component props > Display configuration payload
+ * Querystring parameters from the router > current url params > ClerkJS component props > Display configuration payload
  */
-export const extractAuthProp = (
-  key: ExtractAuthUrlKey,
-  { ctx, queryParams, displayConfig }: ExtractAuthPropOptions,
-): string => {
-  const snakeCaseField = camelToSnake(key);
-  const queryParamValue = queryParams[snakeCaseField];
+export const getRedirectProp = (key: ExtractAuthUrlKey, opts: ExtractAuthPropOptions): string => {
```

Comment:
@nikosdouvlis Let's unit tests as this function has become very critical for the redirection logic.